client/lib/reader-lists-feeds/constants.js

```diff
@@ -0,0 +1,6 @@
+module.exports.action = {
```

Comment:
Aha. I notice `Object.freeze` has been used elsewhere to work around this (e.g. in the importer):

```export const importerTypes = Object.freeze( {
	WORDPRESS: 'importer-type-wordpress',
	GHOST: 'importer-type-ghost',
	MEDIUM: 'importer-type-medium',
	SQUARESPACE: 'importer-type-squarespace'
} );```